source/components/inputs/typeahead/typeahead.html

```diff
@@ -0,0 +1,36 @@
+<div class="field"
```

Comment:
If you add the `.typeahead` class here it will swap out the drop-down caret for a search icon.

![image](https://cloud.githubusercontent.com/assets/13574057/16779114/f208f410-483e-11e6-9573-dce4ed9a4c8d.png)